client/components/social-buttons/google.js

```diff
@@ -110,7 +110,7 @@ class GoogleSocialButton extends Component {
 			}
 		}
 
-		return window.google.accounts.oauth2;
+		return window.google?.accounts?.oauth2;
```

Comment:
NIT: could we null-coalesce this to `null` when optional-chaining fails? We're returning `null` above.

```suggestion
		return window.google?.accounts?.oauth2 ?? null;
```

Reply:
Yeah lets do this! Just to be extra paranoid it makes sense to not introduce any new return type.